services/movie/src/instrumentation.ts

```diff
@@ -24,11 +30,25 @@ const traceExporter = new OTLPTraceExporter({
 
 const spanProcessor = new BatchSpanProcessor(traceExporter);
 
+const provider = new WebTracerProvider({
```

Comment:
Could you remove this provider usage? this variable it not being used byt he NodeSDK, which has its own provider and is probably going to run with no reason on bootstrap